source/ed/connectors/gtfs_parser.cpp

```diff
@@ -400,6 +402,7 @@ nm::StopArea* StopsGtfsHandler::build_stop_area(Data& data, const csv_row& row)
     }
     gtfs_data.stop_area_map[sa->uri] = sa;
     data.stop_areas.push_back(sa);
+    add_gtfs_stop_code(data, sa, row[code_c]);
```

Comment:
You might want to add the stop code only if the column exists (otherwise you'll access `row[-1]` :boom: ) 
```suggestion
if(stop_code_is_present) {
    add_gtfs_stop_code(data, sa, row[code_c]);
}
```